modules/react/button/lib/BaseButton.tsx

```diff
@@ -314,40 +314,36 @@ export const BaseButton = createComponent('button')({
   Component: (
     {
       children,
-      cs,
       size,
       fillIcon,
       iconPosition,
       icon,
+      colors,
```

Comment:
I think we removed this for this reason: https://github.com/Workday/canvas-kit/pull/2381

Reply:
Unless mergeStyles handles this

Reply:
I have no idea why `colors` was removed in #2381. `colors` should be removed from the prop list in `BaseButton` because it isn't a prop that `Box` knew how to handle. It happened to be removed by `Box` because when `Element` was a string because `Box` uses `@emotion/is-prop-valid` and `colors` isn't a valid prop. But if `Element` is not a string, but a component, the `colors` prop would be forwarded to that component which is bad.

My suggestion for the fix was only to delete the line from `Button.tsx` that removed `colors` from `elemProps` passed to `BaseButton`. `colors` should never have been removed from the deconstruction list in `BaseButton`. I'm simply fixing that bug here because I'm no longer using `Box` and no longer using `@emotion/is-prop-valid`